client/app/components/Input/InputTextarea.scss

```diff
@@ -59,3 +59,7 @@
     }
   }
 }
+
+.minHeight{
```

Comment:
Could you rename this to `.editor`?

Reply:
should I replace editor with css.editor then? or keep both editor and css.editor?

Reply:
I think you'll want to keep both actually since we use editor as a reference point in our acceptance tests.

You'll want to do this: 
```
<div className={`editor ${css.editor}`} />
```

Reply:
did the changes. Thank you :D